### ISSUE TITLE
Remaining Time update issue

### ISSUE THREAD

**Issue by almighty059** [dalinicus/homeassistant-acinfinity#84]
I updated to the new 1.8.1 version, which you stated has the breaking change of removing the "Surplus" sensor and replacing it with a "Remaining Time" sensor. However, before I installed the new update, I checked the version I was currently using because I couldn't remember ever using or even seeing a sensor labeled "Surplus." When I checked, I already had a sensor labeled "Remaining Time" and couldn't find any labeled "Surplus." I'm not sure what version I was previously using, but I assume it was the last version you released. Regardless, I still installed the new version, but after doing so, I now have two "Remaining Time" sensors, as you can see in the screenshot below. 

![image](https://github.com/user-attachments/assets/b3987a4d-164f-4caf-99f6-aa1bcb3b23bd)

**Comment by dalinicus** [comment]
The one that says unavailable is the old surplus sensor and can be removed from HA.   The one that's reporting 0 is the new sensor.

The old sensor will show a warning that its no longer being provided, and can be deleted from the cogwheel.

![image](https://github.com/user-attachments/assets/94be6369-4c02-4e66-9617-e2fdb926358a)

![image](https://github.com/user-attachments/assets/91b6c939-c2b3-4b28-b434-c924db428cb3)

**Comment by almighty059** [comment]
Okay. After reading everything I now realize that you created the same sensor but in a different way so that it's compatible with the timer card and can provide the progress. I should have read more before I created the issue. My bad. I actually tried coming up with a template to handle this a while ago but gave. I think I had issues with the way the time was being reported in the integration. I'll have to give it another try. Thanks.
